atd-vzd/migrations/default/1726000511912_fix_timezone_fatalities_view/up.sql

```diff
@@ -0,0 +1,45 @@
+-- fixing a timezone bug in the ytd_fatality and ytd_fatal_crash counts
+create or replace view fatalities_view AS
+select 
+people.id as person_id,
+crashes.id as crash_pk,
+crashes.cris_crash_id,
+crashes.record_locator,
+units.id as unit_id,
+CONCAT_WS(' ', people.prsn_first_name, people.prsn_mid_name, people.prsn_last_name) as victim_name,
+TO_CHAR(crashes.crash_timestamp at time zone 'US/Central', 'yyyy') AS year,
+CONCAT_WS(' ',
+  crashes.rpt_block_num,
+  crashes.rpt_street_pfx,
+  crashes.rpt_street_name,
+  '(',
+  crashes.rpt_sec_block_num,
+  crashes.rpt_sec_street_pfx,
+  crashes.rpt_sec_street_name,
+  ')') AS location,
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'YYYY-MM-DD'
+  ) as crash_date_ct,
+to_char(
+  public.crashes.crash_timestamp at time zone 'US/Central', 'HH24:MI:SS'
+  ) as crash_time_ct,
+-- get ytd fatality, partition by year and sort by date timestamp
+ROW_NUMBER() OVER (
+    PARTITION BY EXTRACT(year FROM crashes.crash_timestamp at time zone 'US/Central') 
+    ORDER BY crashes.crash_timestamp at time zone 'US/Central' ASC) 
```

Comment:
adding this `at time zone 'US/Central'` to these ytd_fatality and ytd_fatal_crash fields is what fixed the bug